### PR TITLE
Copy review

### PR DESCRIPTION
**Pull request by Duncid**
Copy and polish on:
- Assistant Manager
- Assistant Builder
- Help content

### CODE REVIEW

**Review by spolu — DISMISSED** [review]

**Comment by Duncid** [comment]
- @PopDaph Modifying quite heavily Help content
- @fontanierh editing the copy of Assistant

**Comment by Duncid** [comment]
@gabhubert It's a good place to comment & propose changes to the copy.
You can test here:
https://front-edge.dust.tt/w/0ec9852c2f/assistant/new

**Comment by Duncid** [comment]
@PopDaph would be really cool to make the Assistants in the Assistant Gallery clickable (it would insert the @mention in the field).

File: front/pages/w/[wId]/assistant/new.tsx [hunk 1]
```diff
@@ -154,164 +154,159 @@ export default function AssistantNew({
...
-          {/* GETTING STARTED */}
-          <div className="flex flex-col gap-8 pb-32">
-            <div className="mt-6 flex flex-col gap-2">
-              <div className="flex flex-row gap-2">
-                <Icon visual={FlagIcon} size="md" />
-                <span className="text-lg font-bold">Getting started?</span>
-              </div>
-              <p className="text-element-700">
-                Using Assistants is easy as asking a question to a friend or a
+        <div className="s-text-sm s-font-normal s-text-element-800">
```
Comment (spolu) [inline front/pages/w/[wId]/assistant/new.tsx:157]: `s-` classes are not supported in `front` so I guess you can just remove that div?

File: front/pages/w/[wId]/assistant/new.tsx [hunk 2]
```diff
@@ -356,3 +355,28 @@ function StartHelperConversationButton({
     />
   );
 }
+interface Agent {
```
Comment (spolu) [inline front/pages/w/[wId]/assistant/new.tsx:358]: This is a bit heavy to define an interface for such a simple component. Let's just have

```
const AvatarListItem = function({ agent } : { agent : { ... }}) { ... } 
```

**Review by spolu — CHANGES_REQUESTED** [review]
Looks great. Left 2 comments that need addresing :+1:

**Pull request merged**